core/docker/Dockerfile

```diff
@@ -11,7 +11,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM eclipse-temurin:17.0.4_8-jdk
+FROM eclipse-temurin:17-jdk
+
+RUN \
+    set -xeu && \
+    echo 'Acquire::Retries "3";' > /etc/apt/apt.conf.d/80-retries && \
+    echo 'Acquire::http::Timeout "15";' > /etc/apt/apt.conf.d/80-timeouts && \
+    apt-get update -q && \
+    apt-get install -y -q git gcc make && \
```

Comment:
How much larger does this make the image? We shouldn't keep these tools beyond what's needed to build the image.

Reply:
This is a separate builder image. We just copy the (tiny) library into the final image.